src/kernels/jupyter/jupyterDetectionTelemetry.node.ts

```diff
@@ -40,33 +38,6 @@ export class JupyterDetectionTelemetry implements IExtensionSyncActivationServic
         this.globalMemento.update(JupyterDetectionTelemetrySentMementoKey, true).then(noop, noop);
         this.detectJupyter('notebook', process.env).catch(noop);
         this.detectJupyter('lab', process.env).catch(noop);
```

Comment:
aren't we still attempting to send telemetry here? I'm fine with the change, but it doesn't seem like it will resolve the concern raised in the attached issue.

Reply:
oh, the attached issue is closed anyway. So doesn't really matter

Reply:
The original issue is the fact that we're polluting terminal history.
this telemetry sent here is like any other telemetry.